src/coord/src/coord.rs

```diff
@@ -4250,6 +4324,7 @@ where
         for (source_id, _description) in dataflow.source_imports.iter() {
             // Extract `since` information about each source and apply here.
             if let Some(source_since) = self.sources.since_of(source_id) {
+                println!("Source since for {}: {:?}", source_id, source_since);
```

Comment:
@aljoscha Did you mean to merge this line in?

Reply:
Ah dammit, I did not, and thought I had removed id. 🙈